zcash_client_backend/src/data_api/wallet.rs

```diff
@@ -168,8 +168,13 @@ where
 /// };
 ///
 /// let account = AccountId::from(0);
+/// let req = UnifiedAddressRequest {
+///     has_orchard: false,
+///     has_sapling: true,
+///     has_p2pkh: true
+/// };
```

Comment:
This won't compile, because the fields of `UnifiedAddressRequest` are private. I suspect it's not being detected in CI however because of the required feature flags.
```suggestion
/// let req = UnifiedAddressRequest::new(false, true, true);
```
The required feature flags for this doctest will also need to be updated to include `sapling`.